frontend/src/components/Client/PopularProductCard/PopularProductCard.js

```diff
@@ -0,0 +1,38 @@
+import { usePopularProduct } from "../../../hooks/product-hooks"
+import AddToCartButton from "../AddToCartButton/AddToCartButton";
+import PopularProductCardSkeleton from "../Skeletons/PopularProductCardSkeleton/PopularProductCardSkeleton"
+import TextSection from "../TextSection/TextSection";
+
+const PopularProductCard = () => {
+    const { data, status } = usePopularProduct();
+    const popularProduct = data?.data?.data.popularProduct
+    const sampleImage = "https://i.etsystatic.com/24762991/r/il/9f62c0/3430721950/il_794xN.3430721950_4c7c.jpg"
+    if (status === "error") return <h1>Something went wrong!</h1>
+    if (status === "loading") return <PopularProductCardSkeleton />
+    
+    return (
+        <div>
+            <div className="mx-auto max-w-6xl px-6 md:px-2">
+                <div className="flex flex-col md:flex-row md:h-96">
+                    <div className="w-full md:w-96 mr-10">
+                        <img src={sampleImage} className="w-96 h-96 object-cover" />
+                    </div>
```

Comment:
![React App (2)](https://user-images.githubusercontent.com/109117779/229902578-44f328cd-bcff-4074-8d5b-1a7a9b4a7341.gif)

Not sure if it's happening on my computer only, but the image isn't sized the way it appeared yesterday. 
Removing the div around the img (removing 'w-full' and 'md:w-96' as well) and moving the mr-10 into the img tag className seem to fix this weird problem. I don't know if you're having this weird problem too :/